README.md

```diff
@@ -382,6 +382,64 @@ router.route('/pet/:id')
 server.listen(8080)
 ```
 
+## Migrating to 2.x from 1.x
+
+The main change is the update to `path-to-regexp@2.0.0`, which has a few breaking changes:
```

Comment:
The version listed here us different from what is in the history file?

Reply:
Oh, yep must have missed that in the original re-update.  I will take a quick read through that whole commit while I rebase to make sure I didn't miss it anywhere else.